src/model/ident_env.cppo.ml

```diff
@@ -275,7 +313,7 @@ let extract_structure_tree_item item =
     | Tstr_open o ->
       ((extract_extended_open o) :> extracted_items list)
 #endif
-    | Tstr_eval _ | Tstr_value _
+    | Tstr_eval _
     | Tstr_primitive _ | Tstr_typext _
```

Comment:
`Tstr_primitive` binds a value too.

Reply:
This statement isn’t clear

Reply:
`Tstr_primitive` correspond to:
```ocaml
external f : unit -> unit = "%primitive_name"
```
It should be treated as a value (as `val f : unit -> unit`)